tools/dev/dockerpkg.py

```diff
@@ -121,27 +121,6 @@
 """
 
 
-TARGET_DISTROLESS_STAGE = """\
-FROM {base_target_image}
-
-LABEL name="{name}" \\
-      repository="http://github.com/cloud-custodian/cloud-custodian"
-
-COPY --from=build-env /src /src
-COPY --from=build-env /usr/local /usr/local
-COPY --from=build-env /etc/passwd /etc/passwd
-COPY --from=build-env /etc/group /etc/group
-COPY --chown=custodian:custodian --from=build-env /output /output
-COPY --chown=custodian:custodian --from=build-env /home/custodian /home/custodian
-
-USER custodian
-WORKDIR /home/custodian
-ENV LC_ALL="C.UTF-8" LANG="C.UTF-8"
-VOLUME ["/home/custodian"]
-ENTRYPOINT ["{entrypoint}"]
-CMD ["--help"]
-"""
-
```

Comment:
As far as I can tell this is drive-by cleanup following the changes/removals [here](https://github.com/cloud-custodian/cloud-custodian/pull/10140/files#diff-83a6e40e0db5a5d7d6fae0126e2fac8531ba7754d9cf01f859daf73bc43884d4L359-L367)?